File: core-java-practice/leet-code-codebase/areasquare.java
import java.util.*;
public class areasquare {
    public static double calculateArea(double side) {
        return side * side;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double side = sc.nextDouble();
        double area = calculateArea(side);
        System.out.println("Area of square with side " + side + " is: " + area);
    }
}

File: core-csharp-practice/leet-code-codebase/area_square.cs
using System;

class AreaSquare
{
    static void Main()
    {
        Console.Write("Enter the side length of the square: ");
        double side = double.Parse(Console.ReadLine());
        
        double area = side * side;
        
        Console.WriteLine($"Area of the square: {area}");
    }
}